packages/binding-mqtt/src/mqtt.ts

```diff
@@ -47,5 +47,7 @@ export class MqttForm extends Form {
 }
 
 export interface MqttClientConfig {
+    username?: string;
+    password?: string;
```

Comment:
Why? `username` and `password` are already stored inside the credential object.

Reply:
I made it like that so setSecurity method injects user/pass into this MqttClientConfig and this is passed to mqtt.connect(config), so this looks more clear

Reply:
Ok, can you add a comment about this?